focus-ios/focus-ios-tests/XCUITest/SettingTest.swift

```diff
@@ -254,28 +254,30 @@ class SettingTest: BaseTestCase {
         XCTAssertEqual(app.switches["BlockerToggle.Safari"].value! as! String, "0")
 
         iOS_Settings.activate()
-        waitForExistence(iOS_Settings.cells["Safari"])
-        iOS_Settings.cells["Safari"].tap()
-        iOS_Settings.cells["AutoFill"].swipeUp()
-        if #available(iOS 15.0, *) {
-            iOS_Settings.cells.staticTexts["Extensions"].tap()
-        } else {
-            iOS_Settings.cells.staticTexts["CONTENT_BLOCKERS"].tap()
+        if #unavailable(iOS 18) {
+            waitForExistence(iOS_Settings.cells["Safari"])
```

Comment:
On iOS 18, there is no Safari settings available.
![Simulator Screenshot - iPhone 15 - 2024-08-28 at 14 58 45](https://github.com/user-attachments/assets/214e8d03-bdff-47c7-b7ff-bcd90e5acd66)